frontend/app/vite.config.main.ts

```diff
@@ -31,16 +31,8 @@ export default defineConfig({
         'electron',
         'electron-devtools-installer',
         ...builtinModules.flatMap(p => [p, `node:${p}`])
-      ],
-      output: {
-        entryFileNames: '[name].js'
-      }
+      ]
     },
     emptyOutDir: false
-  },
-  css: {
-    modules: {
-      generateScopedName: '[hash:base64:8]'
-    }
```

Comment:
any specific reason why this was removed?

Reply:
the file only job is to build `background.ts` so I think this part is not needed.

Reply:
true, this is only needed for the renderer :)